### ISSUE TITLE
Docker image: persmission denied, several directories

### ISSUE THREAD

**Issue by PureTryOut** [jaedb/Iris#518]
**Describe the bug**

When trying to setup a new Docker container using the `jaedb/iris` image (both with `latest` and `3.44`), it just fails to start on "Permission denied: <some directory>". This happens for multiple directories and I can only get past them by creating them manually, which is quite hard if you can't get into your docker container...

Failing directories I encountered so far:

- `/var/cache/mopidy`
- `/etc/mopidy`

**To Reproduce**
1. On a clean machine, try to run a new `jaedb/iris` Docker image
2. Notice it failing and check it's log

**Expected behavior**
The container to launch successfully obviously

**Environment details:**
 - OS: the Docker hub image `jaedb/iris`

**Comment by jaedb** [comment]
@PureTryOut are you using the recommended Docker Compose setup? Can you please provide the command(s) you're using to start the container?

**Comment by PureTryOut** [comment]
Sorry for the late response.

I'm actually using a self-written Ansible playbook to launch the Docker container, so no not using Docker Compose. However, after coming back to this after a few months, I noticed I was setting those paths in my mopidy.conf. I decided to quit doing manual pathing (not sure why that even worked before) and just used the default conf in this repo, it works now.

Sorry for bothering you, it seems I just needed a fresh look at my setup!
